content/custom_resources/about.md

```diff
@@ -1,66 +1,56 @@
 +++
 title = "Custom Resources"
-draft = false
-
 gh_repo = "chef-web-docs"
-
 aliases = ["/custom_resources.html"]
 
 [menu]
   [menu.infra]
     title = "Custom Resources"
-    identifier = "chef_infra/cookbook_reference/resources/custom_resources.md Custom Resources"
-    parent = "chef_infra/cookbook_reference/resources"
-    weight = 40
+    identifier = "chef_infra/cookbook_reference/custom_resources/custom_resources Custom Resources"
+    parent = "chef_infra/cookbook_reference/custom_resources"
+    weight = 10
 +++
 
 A custom resource:
 
--   Is a simple extension of Chef Infra Client that adds your own resources
--   Is implemented and shipped as part of a cookbook
--   Follows easy, repeatable syntax patterns
--   Effectively leverages resources that are built into Chef Infra Client and/or custom Ruby code
--   Is reusable in the same way as resources that are built into Chef Infra Client
+- Is a simple extension of Chef Infra Client that adds your own resources
+- Is implemented and shipped as part of a cookbook
+- Leverages resources that are built into Chef Infra Client and/or custom Ruby code
+- Is reusable in the same way as resources that are built into Chef Infra Client
 
 For example, Chef Infra Client includes built-in resources to manage
 files, packages, templates, and services, but it does not include a
 resource that manages websites.
 
+## Alternative Reading
+
+Learn Chef interactive tutorial - [Extending Chef Infra: Custom Resources](https://learn.chef.io/courses/course-v1:chef+Infra201+Perpetual/about)
+
 ## Syntax
 
 A custom resource is defined as a Ruby file and is located in a
 cookbook's `/resources` directory. This file:
 
--   Declares the properties of the custom resource
--   Loads current state of properties, if the resource already exists
--   Defines each action the custom resource may take
+- Declares the properties of the custom resource
+- Defines actions the custom resource may take
 
-The syntax for a custom resource is. For example:
+The layout for a custom resource is:
 
 ```ruby
 property :property_name, RubyType, default: 'value'
 
-action :action_name do
+action :action_a do
  # a mix of built-in Chef resources and Ruby
 end
 
-action :another_action_name do
+action :action_b do
  # a mix of built-in Chef resources and Ruby
 end
 ```
 
-where the first action listed is the default action.
-
-{{< warning >}}
-
-Do not use existing keywords from the Chef Infra Client resource system
-in a custom resource, like "name". For example,
-`property :property_name` in the following invalid syntax:
-`property :name, String, default: 'thename'`.
+The first action listed is the default action.
```

Comment:
I feel like this should have a heading around default actions to make it easier to see etc.

Reply:
This is covered in the glossary ##default_action